### PR TITLE
Embedded controlloop2 dev

### PR DESCRIPTION
**Pull request by ezraboley**
Small changes, but a CAN message goes through the entire chain which is big

### CODE REVIEW

File: embedded/drivers/can.c [hunk 1]
```diff
@@ -197,24 +190,20 @@ HAL_StatusTypeDef can_send_intermodule(
...
-	data[6] = 6;
-	data[7] = 7;
 	switch (board) {
 		case DASH:
 			return HAL_ERROR;
 			break;
 		case NAV:
 		    /* Update data, and send out */
             nav_tape_set(data);
+			while( HAL_CAN_GetTxMailboxesFreeLevel( &can_handle ) == 0 ){}
```
Comment (ethanlink17) [inline embedded/drivers/can.c]: I added these for debugging, but don't think they are necessary. We should remove them before we forget

File: embedded/drivers/can.c [hunk 2]
```diff
@@ -197,24 +190,20 @@ HAL_StatusTypeDef can_send_intermodule(
...
             if (can_send_intermodule(NAV, DASH_REC, NAV_TAPE, data) != HAL_OK) 
 				return HAL_ERROR;
             
             nav_should_stop_set(data);
-            if (can_send_intermodule(NAV, ALL, NAV_SHOULD_STOP, data) != HAL_OK) 
+			while( HAL_CAN_GetTxMailboxesFreeLevel( &can_handle ) == 0 ){}
```
Comment (ethanlink17) [inline embedded/drivers/can.c]: Here as well

File: embedded/drivers/can.c [hunk 3]
```diff
@@ -617,7 +608,7 @@ HAL_StatusTypeDef can_init(BOARD_ROLE role) {
 		case DASH:
 			sFilterConfig0.FilterIdHigh		= 0x7ff << 5;
 			sFilterConfig0.FilterIdLow		= 0x0000;
-			sFilterConfig0.FilterMaskIdHigh 	= 0x0000;
+			sFilterConfig0.FilterMaskIdHigh 	= 0x1 << 5;
```
Comment (ethanlink17) [inline embedded/drivers/can.c]: This was also added during debugging and should be changed back to 0x0000

**Comment by rcastle1** [comment]
Looks good to me

**Review by ethanlink17 — APPROVED** [review]

**Pull request merged**